### PR TITLE
Add E-Prime support

### PR DESCRIPTION
**Pull request by Vorror**
E-Prime support for #70 

I did the following things:

- Added e-prime support
- Made e-prime opt-in
- Added --yes-[checkname] support to allow users to enable it
- Modified readme

### CODE REVIEW

**Comment by Vorror** [comment]
Would you guys be ok with a polyfill for Object.assign? (Either npm package or copy-paste job)

**Comment by RichardLitt** [comment]
Add some tests?

**Comment by Vorror** [comment]
Ah, I forgot to add some to write-good itself. I'll add tests later today!

**Comment by Vorror** [comment]
Done :)

**Comment by Vorror** [comment]
@RichardLitt *nudge* ;)

File: write-good.js [hunk 1]
```diff
@@ -49,4 +55,29 @@ function dedup (suggestions) {
   }, []);
 }
 
+// Mozilla Object.Assign Polyfill
+if (typeof Object.assign != 'function') {
```
Comment (RichardLitt) [inline write-good.js]: This is likely a module somewhere. Use this? https://www.npmjs.com/package/object.assign

**Comment by Vorror** [comment]
@RichardLitt  Sure I'll make that change and use the module. In regards to the check, they're all listed [here](https://raw.githubusercontent.com/Vorror/e-prime/master/e-prime.js).

In regards to checking possessive vs contractions. I'm not sure if "sheep's" is a valid contraction of "sheep is", although I very well could be wrong.

**Comment by RichardLitt** [comment]
Well done! Thank you.

**Pull request merged**